pnparser/templates/border.py

```diff
@@ -0,0 +1,59 @@
+from typing import TYPE_CHECKING
+from .common import *
+
+if TYPE_CHECKING:
+    from .pnb import Pnb
+    from .pbc import Pbc
+    from .section import Section
+    from .splash import SplashTableEntry
+
+
+TEMPLATE: str = "{{{{pnbh|context='''{}'''<br>\n"
```

Comment:
this isnt the template right it's the template_start? either way it's not like a huge special global constant so i'd just move it inside of the class

Reply:
Moved all occurrences.